commons/com.b2international.index.tests/src/com/b2international/index/revision/RevisionFixtures.java

```diff
@@ -41,18 +41,18 @@ private RevisionFixtures() {
 		
 		public static class Builder extends Revision.Builder<RevisionData.Builder, RevisionData> {
 
-			protected String id;
-			protected String field1;
-			protected String field2;
-		
+			private String id;
+			private String field1;
+			private String field2;
+			
 			public Builder(RevisionData revisionData) {
 				this.id = revisionData.getId();
 				this.field1 = revisionData.field1;
 				this.field2 = revisionData.field2;
 			}
 
 			public Builder id(String id) {
-				this.id = id;
+				this.id = field1;
```

Comment:
This fix should be kept; it was incorrect before the PR and fixed as part of a commit that has been reverted.

```suggestion
				this.id = id;
```